swagger_server/controllers/itm_ta2_eval_controller.py

```diff
@@ -1,19 +1,21 @@
 import connexion
-import six
+import time
 
 from swagger_server.models.action import Action  # noqa: E501
 from swagger_server.models.alignment_target import AlignmentTarget  # noqa: E501
 from swagger_server.models.scenario import Scenario  # noqa: E501
 from swagger_server.models.state import State  # noqa: E501
-from swagger_server import util
 
 from ..itm import ITMScenarioSession
 
-ITM_SESSION = ITMScenarioSession()
+MAX_SESSIONS = 5     # Hard limit on simultaneously active sessions
+SESSION_TIMEOUT = 60 * 60 * 24  # 24 hour timeout in seconds
+#MAX_SESSIONS = 3     # Hard limit on simultaneously active sessions
+#SESSION_TIMEOUT = 25  # 10 second timeout in seconds
```

Comment:
Do we need these commented out constants?  Look like Dupes.

Reply:
See the PR description/comment.  It was to help with debugging and will be removed, along with some logging.